### PR TITLE
Remove initializing tools status 

### PR DESCRIPTION
**Pull request by Tomorrow9**
For fixing issue #138 remove tools status initialize firstly, then I'll do some changes on this in next release.

### CODE REVIEW

**Comment by Tomorrow9** [comment]
```
VM information:
+-------------------------------------------------------------------------------+
| Name                      | test_vm                                           |
+-------------------------------------------------------------------------------+
| IP                        | xx.xx.xx.xx                                      |
+-------------------------------------------------------------------------------+
| Guest OS Type             | RedHat 8.4 x86_64                                 |
+-------------------------------------------------------------------------------+
| VMTools Version           | 11.3.0.29534 (build-18090558)                     |
+-------------------------------------------------------------------------------+
| CloudInit Version         | 20.3                                              |
+-------------------------------------------------------------------------------+
| Config Guest Id           | rhel8_64Guest                                     |
+-------------------------------------------------------------------------------+
| Hardware Version          | vmx-19                                            |
+-------------------------------------------------------------------------------+
| GuestInfo Guest Id        | rhel8_64Guest                                     |
+-------------------------------------------------------------------------------+
| GuestInfo Guest Full Name | Red Hat Enterprise Linux 8 (64-bit)               |
+-------------------------------------------------------------------------------+
| GuestInfo Guest Family    | linuxGuest                                        |
+-------------------------------------------------------------------------------+
| GuestInfo Detailed Data   | architecture='X86'                                |
|                           | bitness='64'                                      |
|                           | distroName='Red Hat Enterprise Linux'             |
|                           | distroVersion='8.4'                               |
|                           | familyName='Linux'                                |
|                           | kernelVersion='4.18.0-305.el8.x86_64'             |
|                           | prettyName='Red Hat Enterprise Linux 8.4 (Ootpa)' |
+-------------------------------------------------------------------------------+


Test Results (Total: 26, Failed: 2, No Run: 2, Elapsed Time: 02:16:45):
+-------------------------------------------------------------+
| Name                                 |   Status | Exec Time |
+-------------------------------------------------------------+
| deploy_vm_efi_paravirtual_vmxnet3    |   Passed | 00:27:59  |
| ovt_verify_install                   |   Passed | 00:06:38  |
| ovt_verify_status                    |   Passed | 00:03:35  |
| vgauth_check_service                 |   Passed | 00:00:26  |
| check_ip_address                     |   Passed | 00:00:42  |
| stat_balloon                         |   Passed | 00:00:20  |
| stat_hosttime                        |   Passed | 00:00:21  |
| check_inbox_driver                   |   Passed | 00:00:36  |
| check_os_fullname                    |   Passed | 00:00:34  |
| check_efi_firmware                   |   Passed | 00:00:24  |
| cpu_hot_add_basic                    |   Passed | 00:05:42  |
| check_quiesce_snapshot_custom_script |   Passed | 00:01:12  |
| memory_hot_add_basic                 |   Passed | 00:08:28  |
| device_list                          |   Passed | 00:04:16  |
| secureboot_enable_disable            |   Passed | 00:06:38  |
| e1000e_network_device_ops            |   Passed | 00:00:53  |
| vmxnet3_network_device_ops           |   Passed | 00:00:58  |
| cpu_multicores_per_socket            |   Passed | 00:17:48  |
| gosc_perl_dhcp                       | * Failed | 00:07:34  |
| gosc_perl_staticip                   | * Failed | 00:07:16  |
| gosc_cloudinit_dhcp                  |   Passed | 00:04:07  |
| gosc_cloudinit_staticip              |   Passed | 00:04:05  |
| paravirtual_vhba_device_ops          |   Passed | 00:01:52  |
| lsilogic_vhba_device_ops             | * No Run | 00:00:19  |
| lsilogicsas_vhba_device_ops          | * No Run | 00:00:20  |
| ovt_verify_uninstall                 |   Passed | 00:02:18  |
+-------------------------------------------------------------+
```

File: common/vm_get_vmtools_status.yml [hunk 1]
```diff
@@ -58,10 +51,12 @@
...
+  when:
+    - vmtools_is_installed | default(False)
+    - vmtools_not_ready | default(True)
 
 - debug:
     msg:
-      - "VMware tools is installed in guest: {{ vmtools_is_installed }}"
-      - "VMware tools is running in guest: {{ vmtools_is_running }}"
-      - "VMware tools is installed as open-vm-tools: {{ vmtools_is_ovt }}"
+      - "VMware tools is installed in guest: {{ vmtools_is_installed | default(False) }}"
```
Comment (keirazhang) [inline common/vm_get_vmtools_status.yml:60]: Do we need to add checking these 3 variables defined or not in linux or windows test setup and test cases?

**Review by keirazhang — APPROVED** [review]
LGTM

**Comment by Tomorrow9** [comment]
```
VM information:
+------------------------------------------------------------------------------------------+
| Name                      | cycle_win_11_64_pvscsi_efi_ansible                           |
+------------------------------------------------------------------------------------------+
| IP                        | xx.xx.xx.xx                                                |
+------------------------------------------------------------------------------------------+
| Guest OS Type             | Microsoft Windows 11 Enterprise 10.0.22000.0                 |
|                           | 64-bit                                                       |
+------------------------------------------------------------------------------------------+
| VMTools Version           | 12.0.0.31385 (build-18601379)                                |
+------------------------------------------------------------------------------------------+
| Config Guest Id           | windows9_64Guest                                             |
+------------------------------------------------------------------------------------------+
| Hardware Version          | vmx-19                                                       |
+------------------------------------------------------------------------------------------+
| GuestInfo Guest Id        | windows9_64Guest                                             |
+------------------------------------------------------------------------------------------+
| GuestInfo Guest Full Name | Microsoft Windows 10 (64-bit)                                |
+------------------------------------------------------------------------------------------+
| GuestInfo Guest Family    | windowsGuest                                                 |
+------------------------------------------------------------------------------------------+
| GuestInfo Detailed Data   | architecture='X86'                                           |
|                           | bitness='64'                                                 |
|                           | buildNumber='22000'                                          |
|                           | distroName='Windows'                                         |
|                           | distroVersion='10.0'                                         |
|                           | familyName='Windows'                                         |
|                           | kernelVersion='22000.132'                                    |
|                           | prettyName='Windows 10 Enterprise, 64-bit (Build 22000.132)' |
+------------------------------------------------------------------------------------------+


Test Results (Total: 22, Failed: 2, No Run: 1, Elapsed Time: 03:23:55):
+---------------------------------------------------------+
| Name                             |   Status | Exec Time |
+---------------------------------------------------------+
| deploy_vm                        | * No Run | 00:00:00  |
| secureboot_enable_disable        | * Failed | 00:08:49  |
| wintools_complete_install_verify |   Passed | 00:14:38  |
| check_efi_firmware               |   Passed | 00:00:28  |
| check_ip_address                 |   Passed | 00:00:50  |
| check_os_fullname                |   Passed | 00:00:24  |
| mouse_driver_vmtools             |   Passed | 00:00:24  |
| vgauth_check_service             |   Passed | 00:00:21  |
| stat_balloon                     |   Passed | 00:00:21  |
| paravirtual_vhba_device_ops      |   Passed | 00:12:31  |
| lsilogicsas_vhba_device_ops      |   Passed | 00:11:29  |
| sata_vhba_device_ops             |   Passed | 00:10:09  |
| nvme_vhba_device_ops             | * Failed | 00:10:23  |
| e1000e_network_device_ops        |   Passed | 00:08:14  |
| vmxnet3_network_device_ops       |   Passed | 00:06:49  |
| memory_hot_add_basic             |   Passed | 00:47:09  |
| check_quiesce_snapshot           |   Passed | 00:04:19  |
| cpu_multicores_per_socket        |   Passed | 00:19:58  |
| gosc_sanity_staticip             |   Passed | 00:12:11  |
| gosc_sanity_dhcp                 |   Passed | 00:16:52  |
| wintools_uninstall_verify        |   Passed | 00:07:28  |
| cpu_hot_add_basic                |   Passed | 00:05:55  |
+---------------------------------------------------------+
```

**Comment by Tomorrow9** [comment]
No VMware Tools installed in VM:
```
Test Results (Total: 9, Failed: 0, No Run: 7, Elapsed Time: 00:12:06):
+----------------------------------------------------+
| Name                        |   Status | Exec Time |
+----------------------------------------------------+
| deploy_vm                   | * No Run | 00:00:00  |
| secureboot_enable_disable   |   Passed | 00:08:14  |
| check_efi_firmware          |   Passed | 00:00:44  |
| check_ip_address            | * No Run | 00:00:25  |
| check_os_fullname           | * No Run | 00:00:25  |
| mouse_driver_vmtools        | * No Run | 00:00:24  |
| vgauth_check_service        | * No Run | 00:00:24  |
| stat_balloon                | * No Run | 00:00:29  |
| paravirtual_vhba_device_ops | * No Run | 00:00:27  |
+----------------------------------------------------+
```

**Pull request merged**